### ISSUE TITLE
[bug] 0.1.2 f-string issue

### ISSUE THREAD

**Issue by junzheng-wu-whitehorse** [huggingface/smolagents#4]
- version 0.1.2 will crash on a simple import as `from smolagents import CodeAgent`.
- The error log is:
```
  File "/opt/conda/envs/chat/lib/python3.11/site-packages/smolagents/agents.py", line 476
    subtitle=f"{type(self.model).__name__} - {(self.model.model_id if hasattr(self.model, "model_id") else "")}",
```

- Seems like like 476 has a double quote in the f-string:
```
        console.print(
            Panel(
                f"\n[bold]{self.task.strip()}\n",
                title="[bold]New run",
                subtitle=f"{type(self.model).__name__} - {(self.model.model_id if hasattr(self.model, "model_id") else "")}",
                border_style=YELLOW_HEX,
                subtitle_align="left",
            )
        )
```
Should be replaced as:
```
subtitle=f"{type(self.model).__name__} - {(self.model.model_id if hasattr(self.model, 'model_id') else '')}",
```

**Comment by daviden1013** [comment]
0.1.0 has the same issue.

**Comment by aymeric-roucher** [comment]
Thank you for reporting @junzheng-wu-whitehorse!
I didn't get this issue because I was developing in python3.12, where f-strings logic was updated. I just fixed this with [this commit](https://github.com/huggingface/smolagents/commit/329119b716329f3e1cb99f0806ebf50d15286fc6), and published a new pypi release 0.1.3 correcting the issue! Run `pip install smolagents -U`, then you should be good!
